doc/models_library/izhikevich_psc_alpha.rst

```diff
@@ -21,7 +21,7 @@ The dynamics are given by:
    &\;\;\;\; V_m \text{ is set to } c
    &\;\;\;\; U_m \text{ is incremented by } d
 
-On each spike arrival, the membrane potential feels an alpha-shaped current of the form:
+On each spike arrival, the membrane potential feels an alpha-kernel current of the form:
```

Comment:
"feels"? Really?

Reply:
Also, I find "alpha-kernel current" a bit weird.

Reply:
Yes, I noticed the "feels" as well, didn't want to feature creep on this PR, but... that should be addressed.

Compare "alpha-kernel conductance".

Reply:
"Feels" replaced by "is subject to".